apps/mobile/app.json

```diff
@@ -17,12 +16,13 @@
     },
     "assetBundlePatterns": ["**/*"],
     "ios": {
-      "buildNumber": "5",
+      "config": {
+        "usesNonExemptEncryption": false
+      },
```

Comment:
If I don't add this here, I have to manually accept something in AppStoreConnect. This is totally chill.